docs/_static/vunit.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="499.9999999999999" height="499.9999999999999" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I don't think it's necessary to add all the logos here. Having one SVG should be enough (both for the logo and the favicon).